cmd/eno-reconciler/main.go

```diff
@@ -109,10 +104,18 @@ func run() error {
 	}
 
 	writeBuffer := flowcontrol.NewResourceSliceWriteBufferForManager(mgr, writeBatchInterval, 1)
-	err = reconciliation.New(recmgr, writeBuffer, remoteConfig, discoveryMaxRPS, rediscoverWhenNotFound, readinessPollInterval)
+
+	var reconciler reconciliation.Controller
```

Comment:
I don't know if I love this pointer dance 😄 but I also can't think off the top of my head of a better way of doing it, I'll give it some thought though.

Reply:
Good call! Fixed